ui/public/data-user-code-of-conduct-v4.fld/colorschememapping.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>

```

Comment:
The files in the FLD directory and the HTML file were created through this process:
* save GoogleDoc as .docx
* open in Microsoft Word
* save as HTML

See also https://precisionmedicineinitiative.atlassian.net/browse/RW-4977